workitem.go

```diff
@@ -131,11 +131,11 @@ func (c *WorkitemController) Reorder(ctx *app.ReorderWorkitemContext) error {
 		}
 
 		if ctx.Payload.Position.ID == "" {
-			return jsonapi.JSONErrorResponse(ctx, errors.NewBadParameterError("missing position.id", nil))
+			return jsonapi.JSONErrorResponse(ctx, errors.NewBadParameterError("Bad Parameter position.id", nil))
```

Comment:
All error messages should not be capitalized. See here: https://github.com/golang/go/wiki/CodeReviewComments#error-strings